service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/AllergyStructureMapperTest.java

```diff
@@ -70,6 +70,9 @@ public class AllergyStructureMapperTest {
         + "example-allergy-intolerance-resource-20.json";
     private static final String INPUT_JSON_WITHOUT_END_DATE = TEST_FILE_DIRECTORY
         + "example-allergy-intolerance-resource-without-endDate.json";
+
```

Comment:
Tiny change, I know, but please remove the extra space to keep with pattern of the rest of this file.